src/ext/draft/extensible_audio_ports.rs

```diff
@@ -0,0 +1,27 @@
+use crate::plugin::*;
+
+use std::ffi::{c_char, c_void, CStr};
+
+pub const CLAP_EXT_EXTENSIBLE_AUDIO_PORTS: &CStr =
+    unsafe { CStr::from_bytes_with_nul_unchecked(b"clap.extensible-audio-ports.draft0\0") };
+
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
+pub struct clap_plugin_extensible_audio_ports {
+    pub add_port: Option<
+        unsafe extern "C" fn(
+            plugin: *const clap_plugin,
+            is_input: bool,
+            channel_count: u32,
+            port_type: *const c_char,
+            port_details: *const c_void,
+        ) -> bool,
+    >,
+    pub remove_port: Option<
+        unsafe extern "C" fn(
+            plugin: *const clap_plugin,
+            is_input: bool,
+            channel_count: u32,
```

Comment:
Should be `index: u32`: https://github.com/free-audio/clap/blob/41964fa3f44fdbf6feb20df528a9cdf2cc3b05ba/include/clap/ext/draft/extensible-audio-ports.h#L28